templates/page/grande-marche-europe/grande-marche-europe.html.twig

```diff
@@ -0,0 +1,87 @@
+{% block canonical_url url('page_grande_marche_europe') %} 
+{% block header '' %}
+{%block banner '' %}
+{% block javascripts %} {{ parent() }}
+<script type="text/javascript">
+Kernel.onLoad(function() {
+    App.runGrandeMarcheEurope();
+});
+</script>
+{% endblock %} 
+{% block content %}
+<main class="content grande-marche-europe">
+    <section id="splash-screen">
+       <div class="logo text--white text--center ">La Grande Marche</div>
+    </section>
+        <section class="video--bkg">
+            <header>
+                <div class="logo text--white ">La Grande Marche</div>
+                <div class="socials text--white">
+                  Partager sur :
+                <a href="#"><i class="fa fa-lg fa-facebook-square "></i></a>
+                <a href="#"><i class="fa fa-lg fa-twitter "></i></a>
+                </div>
+            </header>
+            <h1 classa="text--white text--large text--center">
+                Rejoignez <br>
+                la grande Marche de l'Europe</h1>
+            <div class="counter">
+                <div class="counter-unit">
+                    <div>
+                        <span id="day"></span>
+                        <p>Jours</p>
+                    </div>
+                    <div>
+                        <span id="hour"></span>
+                        <p>Heures</p>
+                    </div>
+                    <div>
+                        <span id="minute"></span>
+                        <p>Minutes</p>
+                    </div>
+                    <div>
+                        <span id="second"></span>
+                        <p>Secondes</p>
+                    </div>
+                </div>
+            </div>
+            <div class="filter"></div>
+             <video src="./videos/gMarche.mp4" autoplay loop></video> 
+        </section>
+        <div class="testimonial--content">
+            <div class="quote">
+                <div class="quote--content">
+                    <p class="text--italic text--center text--bold">
+                        « L’Europe fait partie de l’ADN de La République En Marche : 
+                        nous portons un engagement pro-européen sans ambigüité. 
+                        Il est assumé mais exigeant !
+                        C’est pourquoi nous partons dès à présent à l’écoute de nos concitoyens, 
+                        afin de réinventer avec eux notre projet pour l’Europe. »
+                        <br>
+                    </p>
+                    <p class="quote--author text--center text--white text--uppercase-wide text--normal">
+                        Christophe Castaner, Délégué général de LaREM
+                    </p>
+                </div>
+            </div>
+            <div class="testimonial--img text--center">
+                <div class="img--castaner progressive-background" 
+                data-sd="{{ static_asset('castaner.jpg', {'w': 300, 'q': 20}) }}"
```

Comment:
simple quotes in array keys are useless and you should add white spaces around arrays content : 
`{{ static_asset('castaner.jpg', { w: 300, q: 20 }) }}`